src/main/java/org/quiltmc/loader/impl/transformer/QuiltTransformer.java

```diff
@@ -44,7 +44,7 @@
 	}
 
 	public static byte[] transform(boolean isDevelopment, EnvType envType, String name, byte[] bytes) {
-		boolean isMinecraftClass = name.startsWith("net.minecraft.") || name.indexOf('.') < 0;
+		boolean isMinecraftClass = name.startsWith("net.minecraft.") || name.startsWith("com.mojang.blaze3d.") || name.indexOf('.') < 0;
```

Comment:
May be worth considering a better way to handle this in the future. Throw a fixme on it